### PR TITLE
profile part is created but no made dynamically

### PR DESCRIPTION
**Pull request by PennCreative**
I filled in the  profile section with some filler content. It's not being pushed dynamically into the div. This altered the "utilities" and I added a "border-radius" to the css.

### CODE REVIEW

**Review by shalane-proctor — APPROVED** [review]
I love the fact the beard man changes at random! Very funny!

**Review by scamp925 — COMMENTED** [review]
Love the look and rotating profile pictures are great! Before I approve, I just want to ask if the list items in the unordered list will be links? Or for this project, will they be "for decoration"? This question is for anyone to answer.

**Review by scamp925 — APPROVED** [review]

**Pull request merged**
